modules/theme/src/themes/default/collections/menu.overrides

```diff
@@ -274,6 +274,12 @@
                     }
                 }
             }
+            // Fix for username and avatar in the header being pushed right to the right edge in tablet view
```

Comment:
You might have to write a media query for this. Otherwise in the desktop viewports avatar will be slightly aligned to the left.

<img width="346" alt="Screen Shot 2020-02-06 at 12 40 25 PM" src="https://user-images.githubusercontent.com/25959096/73913859-14600c80-48de-11ea-9288-cea490965ae8.png">

Reply:
It's already inside a media query.
<img width="813" alt="Screenshot 2020-02-06 at 12 50 48" src="https://user-images.githubusercontent.com/20745428/73914498-6a817f80-48df-11ea-8291-3ecb61439039.png">